stacks/filecoin-stack.js

```diff
@@ -121,6 +121,7 @@ export function FilecoinStack({ stack, app }) {
           AGGREGATOR_DID,
           PROOF: STOREFRONT_PROOF,
         },
+        timeout: '4 minutes',
```

Comment:
is there some maths or experiments to back this up, or just bump it up an see?

Reply:
we run each 5 minutes, the amount of work to be done depends on the inflow of uploads. Timing out with 10 seconds would not work when we may need to get 1000s of receipts and do 1000s of dynamo ops. Timing out after 4min and restarting the job after will restart the process and will be fine, so there is not really math behind this